kubernetes/samples/charts/voyager/README.md

```diff
@@ -21,7 +21,7 @@ $ helm search repo appscode/voyager
 NAME               CHART VERSION APP VERSION	DESCRIPTION
 appscode/voyager   v12.0.0       v12.0.0    	Voyager by AppsCode - Secure HAProxy Ingress Co...
 ```
-> **NOTE**: After updating the helm repository, the Voyager version listed maybe newer that the one appearing here, please check with the Voyager site for the lastest supported versions.
+> **NOTE**: After updating the helm repository, the Voyager version listed maybe newer that the one appearing here, please check with the Voyager site for the latest supported versions.
 
 ### 2. Install the Voyager operator
```

Comment:
Wait until the Voyager Operator is running. -> Wait until the Voyager operator is running. (You do not capitalize operator elsewhere, don't do it here on line 38.)
Please change ALL instances of "Ingress" (capital I) to ingress (lower case).

Reply:
done